### PR TITLE
Publish node task runner

### PR DESCRIPTION
**Pull request by BMurri**
Addresses #265

### CODE REVIEW

**Comment by BMurri** [comment]
Before closing the issue, I'll be adding build instructions to both wikis.

File: src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        <Private>False</Private>
+    </ProjectReference>
+    <None Condition=" '$(TesApiWebArchive)' != '' "
+          Include="$(TesApiWebArchive)\tesapi.dll"
+          MsBuildSourceProjectFile="$(TesApiWebArchive)\proj"
+          OutputPath="tesapi.dll"
+          TargetPath="tesapi.dll" />
+  </ItemGroup>
+
+  <!--<Target Name="__AssignPublishProperties" BeforeTargets="ResolveProjectReferences" AfterTargets="PrepareProjectReferences">
```
Comment (giventocode) [inline src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj:40]: Can this commented out code be removed?

File: src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            var dirs = new List<ITaskItem>();
+            foreach (var file in PublishedFiles)
+            {
+                Log.LogMessage(MessageImportance.Normal, "Parsing {0}", file.ItemSpec);
+                var dirInfo = Directory.GetParent(Path.Combine(Directory.GetParent(file.GetMetadata("MsBuildSourceProjectFile")).FullName, file.GetMetadata("OutputPath")));
+                var dir = new TaskItem(dirInfo.FullName);
+                file.CopyMetadataTo(dir);
+                dirs.Add(dir);
+            }
+            //PublishedDirectories = Enumerable.DistinctBy(dirs, item => item.ItemSpec).ToArray();
```
Comment (giventocode) [inline src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj:80]: Same as above, commented out code should be removed?

**Review by giventocode — CHANGES_REQUESTED** [review]

File: src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj (hunk 2 above)
Comment (BMurri) [reply src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj]: Done

File: src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj (hunk 1 above)
Comment (BMurri) [reply src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj]: Done

**Pull request merged**